omnibus/.kitchen.yml

```diff
@@ -18,6 +18,13 @@ driver:
 provisioner:
   name: chef_zero
   require_chef_omnibus: true # Always install the latest version of Chef
+  attributes:
+    vagrant:
+      this_key_exists_so_we_have_a_vagrant_key: true
```

Comment:
I am not entirely sure how this is used -- but Test Kitchen exports the shell environment variable `TEST_KITCHEN=1` as well.